template/scripts/drupal-vm/config.yml

```diff
@@ -0,0 +1,30 @@
+# Update the hostname to the local development environment hostname
+vagrant_hostname: ${project.machine_name}.localhost
```

Comment:
For some reason I feel like this should be ${project.machine_name}.vm rather then .localhost